src/lib/getTxLink.ts

```diff
@@ -13,8 +16,25 @@ type GetTxBaseUrl = {
   isOrder?: boolean
 }
 
+// An eip-3770 compliant mapping of ChainId to chain shortname
+// https://github.com/safe-global/safe-core-sdk/blob/ea0d5018a93f294dfd891e6c8963edcb96431876/packages/protocol-kit/src/utils/eip-3770/config.ts
+export const chainShortnameByChainId: Partial<Record<KnownChainIds, string>> = {
```

Comment:
nitpick - explicitely name this variable `eip3770ChainShortNameByChainId` or even `safeShortNameByChainId`

Reply:
how about 25f32077f6